apps/webapp/app/components/runs/v3/RunFilters.tsx

```
@@ -914,12 +977,549 @@ function AppliedPeriodFilter() {
+        />
+      )}
+    </FilterMenuProvider>
+  );
+}
+
+function dateFromString(value: string | undefined | null): Date | undefined {
+  if (!value) return;
+
+  //is it an int?
+  const int = parseInt(value);
+  if (!isNaN(int)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)